### DISCUSSION TITLE
Progetto esame Cristiano Moscetti Castellani

### DISCUSSION THREAD

**Discussion by Meuritz** [deneb-alpha/Lab_open_source-2023_2024#23] [Projects]
Come fare "Bug Finding?" (Guida su come trovare/confermare bug)

1. Che cosa è un Bug?
2. Virtual Machines
3. Windows / Linux
4. Bugzilla ODF
5. alcuni bug(bug che ho confermato)

EXTRA:
provare a confermare un bug in live(non so come potrebbe venire dato che siamo in call)

**Comment by deneb-alpha** [comment]
Ciao @Meuritz,
come ho suggerito anche ad altri, mi piacerebbe anche vedere qualcosa di più pratico su git.
Se non dovessi aver modo di lavorare alle traduzioni di LibreOffice, ti suggerirei di creare un repo personale in cui caricare la tua presentazione e in cui documentare gli step necessari per farlo, dalla creazione del repo fino al caricamento delle slide.
Non dimenticarti di utilizzare REUSE ;)

**Reply by Meuritz** [reply]
Ho cambiato completamente il progetto, non so se la terminologia che ho usato è adatta

**Reply by deneb-alpha** [reply]
Ciao @Meuritz :) 
non ti vedo iscritto all'appello di domani.

Comunque, in generale la proposta  di progetto può  andar bene.
* Pensi di parlare anche del bisect? 
* Nei punti 2 e 3 a cosa  fai riferimento esattamente?

Per la presentazione non preoccuaprti, puoi  fare condivisione dello schermo e, mentre fai il triage,  puoi  spiegare  cosa fai, i ragionamenti seguiti etc...

**Reply by Meuritz** [reply]
2 e 3 sono "i preparativi" per fare triage, dato che magari qualche bug potrebbe essere specifico su un sistema operativo, volevo mettere anche Mac OS X, ma per virtualizzarlo serve una scheda video AMD.. e un mac da cui prendere il seriale

**Reply by Meuritz** [reply]
per quanto riguarda l'esame, lo do al prossimo appello ovvero il 5 febbraio
